src/v2/controllers/FormController.js

```diff
@@ -46,6 +48,22 @@ export default Controller.extend({
       currentViewState.name,
       this.options.appState.get('authenticatorKey'),
     );
+
+    // Walk through every authenticator and apply customizations
+    this.options.appState
+      .getRemediationAuthenticationOptions(currentViewState.name)
```

Comment:
This make sense only for the select authenticator view. Other views which are individual enroll or challenge views need not run this for all authenticators I guess. 

This works but I guess we can do better here. Will keep thinking.